lib/classes/Error.js

```diff
@@ -42,67 +58,69 @@ module.exports.ServerlessError = class ServerlessError extends Error {
-
-    consoleLog(chalk.yellow('  Get Support --------------------------------------------'));
-    consoleLog(`${chalk.yellow('     Docs:          ')}${'docs.serverless.com'}`);
-    consoleLog(
-      `${chalk.yellow('     Bugs:          ')}${'github.com/serverless/serverless/issues'}`
-    );
-    consoleLog(`${chalk.yellow('     Issues:        ')}${'forum.serverless.com'}`);
+module.exports.logError = (exception, { forceExit = false } = {}) => {
+  const exceptionMeta = resolveExceptionMeta(exception);
+  const isUserError = exceptionMeta.name === 'ServerlessError';
+
+  const possiblyExit = () => {
+    if (forceExit) process.exit();
+  };
+
+  writeMessage(exceptionMeta);
+
+  if (!isUserError && !process.env.SLS_DEBUG) {
+    const debugInfo = [
+      '    ',
+      ' For debugging logs, run again after setting the',
+      ' "SLS_DEBUG=*" environment variable.',
+    ].join('');
+    consoleLog(chalk.red(debugInfo));
+    consoleLog(' ');
+  }
 
+  if (exceptionMeta.stack && (!isUserError || process.env.SLS_DEBUG)) {
+    consoleLog(chalk.yellow('  Stack Trace --------------------------------------------'));
     consoleLog(' ');
-    consoleLog(chalk.yellow('  Your Environment Information ---------------------------'));
-    consoleLog(chalk.yellow(`     Operating System:          ${platform}`));
-    consoleLog(chalk.yellow(`     Node Version:              ${nodeVersion}`));
-    consoleLog(chalk.yellow(`     Serverless Version:        ${slsVersion}`));
-    consoleLog(chalk.yellow(`     Enterprise Plugin Version: ${sfeVersion}`));
-    consoleLog(chalk.yellow(`     Platform SDK Version:      ${sdkVersion}`));
+    consoleLog(exceptionMeta.stack);
     consoleLog(' ');
+  }
+
+  const platform = process.platform;
+  const nodeVersion = process.version.replace(/^[v|V]/, '');
+  const slsVersion = version;
 
-    process.exitCode = 1;
-    // Exit early for users who have opted out of tracking
-    if (!errorReporter.installed) return;
-    // report error to sentry.
-    errorReporter.captureException(e, () => {});
-  } catch (errorHandlingError) {
-    throw new Error(inspect(e));
+  consoleLog(chalk.yellow('  Get Support --------------------------------------------'));
+  consoleLog(`${chalk.yellow('     Docs:          ')}${'docs.serverless.com'}`);
+  consoleLog(`${chalk.yellow('     Bugs:          ')}${'github.com/serverless/serverless/issues'}`);
+  consoleLog(`${chalk.yellow('     Issues:        ')}${'forum.serverless.com'}`);
+
+  consoleLog(' ');
+  consoleLog(chalk.yellow('  Your Environment Information ---------------------------'));
+  consoleLog(chalk.yellow(`     Operating System:          ${platform}`));
+  consoleLog(chalk.yellow(`     Node Version:              ${nodeVersion}`));
+  consoleLog(chalk.yellow(`     Serverless Version:        ${slsVersion}`));
+  consoleLog(chalk.yellow(`     Enterprise Plugin Version: ${sfeVersion}`));
+  consoleLog(chalk.yellow(`     Platform SDK Version:      ${sdkVersion}`));
```

Comment:
We just changed the name of those values via https://github.com/serverless/serverless/pull/6501 so this PR might need a rebase / update.

Reply:
Actually it was not changed here (in master it's still `Enterprise Plugin Version`). I can rename it, still isn't `Plugin` name too vague?

Reply:
Yes, most agreed that it's too vague. According to JIRA that's the whole idea (vague == more flexible)